src/tslint-cli.ts

```diff
@@ -22,9 +22,9 @@ const fs = require("fs");
 const optimist = require("optimist")
     .usage("usage: $0")
     .check((argv: any) => {
-        // at least one of file or help or version must be present
-        if (!(argv.f || argv.h || argv.v)) {
-            throw "Missing required arguments: f";
+        // at least one of file, help, version or unqualified argument must be present
```

Comment:
If we're removing `-f` altogether, let's add a deprecation notice here?
Check for `argv.f` and `throw "-f option is deprecated. Supply files directly to the tslint command instead."`